docs/markdown/snippets/int_to_string_padding.md

```diff
@@ -0,0 +1,17 @@
+## Added 'fill' kwarg to int.to_string()
+
+int.to_string() now accepts a `fill` argument. This allows you to pad the
+string representation of the integer with leading zeroes:
+
+```meson
+message(4.to_string())
+message(4.to_string(length: 3))
+message(4.to_string(length: -3))
```

Comment:
Sorry, I haven't been clear. I meant that case:
```
>>> str(-4).zfill(3)
'-04'
```
Notice how it did not blindly left filled with zeroes, it padded after the sign char and did count it for the lenght of the string.

Reply:
https://docs.python.org/3/library/stdtypes.html#str.zfill

Reply:
Works with `+` sign as well in python, but in Meson to_string() won't add the sign for positive numbers.

```
>>> '+4'.zfill(3)
'+04'
```